cypress/integration/templates.ts

```diff
@@ -461,6 +461,30 @@ context('Template tests', () => {
       cy.contains(textQuestion.newId);
       /* --- */
 
+      /* Check if template preview works */
+      cy.get('[data-cy="preview-questionary-template"]').click();
+      cy.get('[aria-labelledby="preview-questionary-template-modal"]').should(
+        'exist'
+      );
+
+      cy.get('[data-cy="questionary-stepper"] button').last().click();
+
+      cy.get(
+        '[aria-labelledby="preview-questionary-template-modal"] form'
```

Comment:
I wonder if we can use data-cy property instead of aria-labelledby or if you think using aria-* labels is fine. Not sure.

Reply:
Using data-cy property on the Dialog component is not working. The only solution to this is to wrap it in a `div` or some other proper element and add data-cy there (which to be honest I don't like much). I would prefer to keep it like this for now and if there is some general solution to all modals to be able to add `data-cy` prop would be nice for later usage.